frontend/src/App.jsx

```diff
@@ -18,7 +18,9 @@ class App extends React.Component {
         return (
             <React.Fragment>
                 <AdminHeader />
-                <AdminRoutes />
+                <div className="view-container">
+                    <AdminRoutes />
```

Comment:
does this not work if you do `<AdminRoutes className="view-container" />`?

Reply:
I believe `AdminRoutes` returns a `React.Fragment`, so it cannot have a class name.